Corrects gort arduino upload command

lcguida opened the pull request:
`gort` command should be `gort arduino upload firmdata <port>` instead of `gort arduino upload <port>`, as stated here http://artoo.io/documentation/getting-started/tutorials/arduino/ and here https://github.com/hybridgroup/gort/issues/38. Following this tutorial leads the user to an error.

Commit:
Update arduino.html.haml

`gort` command should be `gort arduino upload firmdata <port>`, as stated here http://artoo.io/documentation/getting-started/tutorials/arduino/ and here https://github.com/hybridgroup/gort/issues/38. Following this tutorial leads the user to an error.

deadprogram commented on source/documentation/platforms/arduino.html.haml:
```diff
@@ -88,7 +88,7 @@ subnavjs: true
     Use the serialport address we found with the earlier `gort`, or leave it blank to use the default address (`/dev/ttyACM0`).
 
-        $ gort arduino upload /dev/ttyACM0
+        $ gort arduino upload firmdata /dev/ttyACM0
```
Thanks for the fix. However, I think this should be:

```
$ gort arduino upload firmata /dev/ttyACM0
```

lcguida replied:
Done.

Commit:
Correcting gort command

As @deadprogram pointed out, the command should be `firmata`
and not `firmdata`.

deadprogram commented:
Thank you, @lcguida

lcguida commented:
Glad to help. Great work by the way. Real fun work on robotics with ruby.
